blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,191 @@
+
+## AKS LTS Version Compatibility
+
+**Every currently supported AKS Kubernetes version is now also available for long term support (LTS).** This means you can immediately access long-term support for your existing clusters without requiring cluster upgrades or migrations.
+
+Whether you're running the latest version or an older supported version, you can transition to LTS support coverage today by upgrading your cluster to the Premium tier.
+
+**Immediate Benefits:**
+- **No forced migrations:** Your existing clusters can adopt LTS support in-place
+- **Complete coverage:** Every supported version gets the same comprehensive LTS treatment
+- **Instant value:** Start benefiting from extended support timelines immediately
+
+When planning your LTS adoption, consider:
+- **Current cluster assessment:** Evaluate which of your existing clusters would benefit most from extended support
+- **Workload criticality:** Prioritize mission-critical production workloads for LTS coverage
+- **Compliance requirements:** Align LTS adoption with your organization's governance and compliance needs
+
+For detailed migration guidance and best practices, refer to our [AKS LTS migration documentation](https://aka.ms/aks/lts-migration) on Microsoft Learn.
+
+## Upgrading Between LTS Versions
+
+When it's time to upgrade from one AKS LTS version to another, you can take advantage of all the robust upgrade functionality already available in AKS. Popular upgrade options include:
+
+- **MaxUnavailable configuration:** Control cluster capacity during upgrades by specifying the maximum number of nodes that can be unavailable simultaneously
+- **Undrainable node behavior:** Configure how AKS handles nodes that cannot be drained, ensuring predictable upgrade outcomes
+- **OS Security Patch channel:** Automate operating system security updates through configurable patch channels
+- **Node Image channel:** Keep node images updated with the latest security patches and OS improvements
+- **Planned maintenance windows:** Schedule upgrades during off-peak hours to minimize business impact
+
+For comprehensive guidance on all available upgrade strategies, see the [AKS cluster upgrade documentation](https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster). These production-tested upgrade mechanisms ensure smooth transitions between LTS versions while maintaining workload availability.
+
+## Looking Ahead
+
+AKS LTS represents our commitment to supporting enterprises at every stage of their Kubernetes journey. While LTS focuses on stability, we continue to innovate rapidly in standard AKS, ensuring you have access to the latest Kubernetes capabilities when you need them.
+
+Over the coming months, we'll be expanding AKS LTS capabilities based on your feedback, including:
+
+- **Istio support for LTS:** Bringing comprehensive LTS support for Istio service mesh to provide enterprise-grade stability for your microservices architecture
```

Comment:
is Istio the only limitation? everything else already under LTS coverage? let's chat internally on this

Reply:
cilium and KMS V2 we should be good.

Reply:
KMS v2 is not supported with LTS as of now (I can confirm that). We are planning enhancements to the current KMS v2 experience, which are tentatively expected in preview CY2025Q4 and GA CY2026Q1. once the improved experience (which is based on a new design) is GA, then KMS v2 will be available with LTS.